tile_generator/helm.py

```diff
@@ -0,0 +1,41 @@
+import os
+import sys
+import yaml
+import json
+
+def find_required_images(values):
+    images = []
+    values = { k.lower():v for k,v in values.items() }
```

Comment:
Since you are doing a `for key, value in values.items()` below already we can just put `key = key.lower()` on line 10 and save our selves a loop.

Reply:
Resolved